libs/sdk-ui-charts/styles/scss/headline.scss

```diff
@@ -220,3 +220,15 @@
         }
     }
 }
+
+.gd-visually-hidden {
```

Comment:
we already have sr-only class for this in ui-kit, check if it is not available in ui-charts too